### PR TITLE
Urlhaus add reliability

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: demisto/etc#29347

## Description
Added reliability

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by JasBeilin** [comment]
Add changes to the readme.
Lets take another look on the testing, if not lets add a picture of the integration's instance with the change.

File: Packs/URLHaus/Integrations/URLHaus/URLHaus.py [hunk 1]
```diff
@@ -495,22 +475,51 @@ def urlhaus_download_sample_command():
...
+
+def main():
+    try:
+        params = demisto.params()
+
+        args = {
+            'api_url': params['url'].rstrip('/'),
+            'use_ssl': not params.get('insecure', False),
+            'threshold': int(params.get('threshold', 1))
+        }
```
Comment (bakatzir) [inline Packs/URLHaus/Integrations/URLHaus/URLHaus.py:487]: please separate args from params.

File: Packs/URLHaus/Integrations/URLHaus/URLHaus.py [hunk 2]
```diff
@@ -495,22 +475,51 @@ def urlhaus_download_sample_command():
...
+            args['reliability'] = DBotScoreReliability.get_dbot_score_reliability_from_str(reliability)
+        else:
+            Exception("Please provide a valid value for the Source Reliability parameter.")
+
+        # Remove proxy if not set to true in params
+        if not demisto.params().get('proxy'):
+            os.environ.pop('HTTP_PROXY', None)
+            os.environ.pop('HTTPS_PROXY', None)
+            os.environ.pop('http_proxy', None)
+            os.environ.pop('https_proxy', None)
```
Comment (bakatzir) [inline Packs/URLHaus/Integrations/URLHaus/URLHaus.py:502]: you can use `handle_proxy` instead.

File: Packs/URLHaus/Integrations/URLHaus/URLHaus_test.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import requests
+import demistomock as demisto
+import json
+from CommonServerPython import DBotScoreReliability
+import URLHaus
+
+
+def test_reliability_in_dbot(mocker):
```
Comment (bakatzir) [inline Packs/URLHaus/Integrations/URLHaus/URLHaus_test.py:8]: please use when/given/then

**Review by bakatzir — CHANGES_REQUESTED** [review]

File: Packs/URLHaus/Integrations/URLHaus/URLHaus.py (hunk 1 above)
Comment (yaakovpraisler) [reply Packs/URLHaus/Integrations/URLHaus/URLHaus.py:487]: done.

File: Packs/URLHaus/Integrations/URLHaus/URLHaus.py (hunk 2 above)
Comment (yaakovpraisler) [reply Packs/URLHaus/Integrations/URLHaus/URLHaus.py:502]: done.

File: Packs/URLHaus/Integrations/URLHaus/URLHaus_test.py (hunk 3 above)
Comment (yaakovpraisler) [reply Packs/URLHaus/Integrations/URLHaus/URLHaus_test.py:8]: done.

**Review by bakatzir — APPROVED** [review]

**Pull request merged**